package.json

```diff
@@ -25,6 +25,7 @@
 		"cli-app",
 		"cli",
 		"chrome",
+		"brave",
```

Comment:
I just thought adding this may increase discoverability. Not really committed to the idea.